caasp-multi-master-stack.yaml

```diff
@@ -265,9 +320,37 @@ resources:
             runcmd:
               - /usr/bin/systemctl enable ntpd
               - /usr/bin/systemctl start ntpd
+              - mkdir /etc/caasp/cpi
+              - chmod -R 0777 /etc/caasp/cpi
```

Comment:
Why? Can this be made slightly less insecure?

Reply:
Looking into these permissions. 

All the comments by @mjura were addressed already,  marked them as "done outdated"

Reply:
now set permissions to 0755 on the /etc/caasp/cpi directory (same as other dirs in that caasp folder) and 0644 on /etc/caasp/cpi/openstack.conf